lib/plottr_components/src/components/timeline/AddLineRow.js

```diff
@@ -192,6 +192,7 @@ const AddLineRowConnector = (connector) => {
       })
     }
 
+    // TODO: refactor by putting this logic in reducers
```

Comment:
Ah!  I see that you're already thinking about adding plot lines from templates :)

Another benefit of moving this all to the reducers is that the mobile guys won't have to re-implement it.

Reply:
YES!